### ISSUE TITLE
Cannot see the environment in React360 inside OTPublisher/ OTSubscriber

### ISSUE THREAD

**Issue by zaidSimajji** [opentok/opentok-react-native#183]
# Bug Report

**Current behavior**
I'm trying to use opentok and react 360 together to create an app where user A and B can explore/ move around in  react 360 enviroment (using react native's webview) while seeing each other's screen using opentok. However the OTPublisher and OTSubscriber only shows a blank screen except the react 360's camera direction indicator. The camera's direction indicator in OTPublisher and OTSubscriber does move when I move the camera in react 360.

**Steps to reproduce**
1. Create a new project
2. Install opentok-react-native plugin
3. Create a screen that shows  react 360 through webview (the HTML file is on a server)
4. Added OTSession, OTPublisher, and OTSubscriber into the screen and set the videoSource to 'screen'.
5. Run on android device, the screen shows react 360 environment but the OTPublisher and OTSubscriber doesn't show anything other than the camera's direction indicator.

**What is the current *bug* behavior?**
The screen now shows react 360 enviroment but the OTPublisher and OTSubscriber doesn't show anything other than the camera's direction indicator.

**What is the expected *correct* behavior?**
The OTPublisher and OTSubscriber should show what the user sees on their screen as well as the other user's screen. 

**Relevant logs and/or screenshots**
Below is the screenshots of the screen (the blank white box is the OTPublisher):
![screenshot_2018-11-02-11-14-23-422_com rn360hybrid](https://user-images.githubusercontent.com/30492895/47892609-a8e5ee80-de93-11e8-8790-05534ddba515.png)

**Comment by zaidSimajji** [comment]
After some digging and testing with various websites, I noticed that OpenTok will not show anything that is in WebGL. According to people at React 360; the environment is also done in WebGL while the indicator is a normal HTML element, which would explain the situation I have. 

Does anybody know if this is the case or perhaps there's a way to actually do this? I've tried searching about in Google but didn't see anything that is  related to it.
